usr/bin/nxdk-c++

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+
+. ${NXDK_DIR}/usr/share/nxdk/prelude.inl
+
+if [ "$OS" = "macos" ]; then
+  CLANG="/usr/local/opt/llvm/bin/clang"
+else
+  CLANG="clang"
```

Comment:
Just out of curiosity, why not: 
```sh
CLANG="clang"

if [ "$OS" = "macos" ]; then
   CLANG="/usr/local/opt/llvm/bin/clang"
fi
```

Reply:
No particular reason. I wanted to respond that I liked mine better, but failed to find arguments.
So I'll probably adopt yours.